apps/backoffice-v2/src/pages/Entities/hooks/useEntities/useEntities.tsx

```diff
@@ -94,6 +94,7 @@ export const useEntities = () => {
     onSortDirToggle,
     search,
     cases: searched,
+    caseCounter: data?.meta?.totalItems,
```

Comment:
`count` makes more sense than `counter`. It's a more common convention
```suggestion
    caseCount: data?.meta?.totalItems,
```